SplunkRumCrashReporting/SplunkRumCrashReportingTests/DeviceStatsTests.swift

```diff
@@ -0,0 +1,35 @@
+//
+/*
+Copyright 2021 Splunk Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+@testable import SplunkRumCrashReporting
+import Foundation
+import XCTest
+
+class DeviceStatsTests: XCTestCase {
+    func testBattery() throws {
```

Comment:
Maybe a bit stronger test for these values besides `!.isEmpty` ? - perhaps "should contain a number or be `Unknown`"

Reply:
I changed these tests around to test for more than the presence of a value.